### DISCUSSION TITLE
Non blurred background on person picture

### DISCUSSION THREAD

**Discussion by etibones** [lllyasviel/Fooocus#1093] [General]
Hello, is there a way to make the background less blurry so it looks more realistic?

**Comment by hesi18** [comment]
?

**Comment by ivancollider** [comment]
I agree with you, same problem, we would need something that helps.

The only thing that currently works for me is to put the sharpness at 10 and the Guidance Scale at 7 in the advanced options, then go to Outpainting (Default) and paint where the background is. If instead in Outpainting you use Modify content, putting the Guidance Scale at 10 helps even more, these are the tests I have done, don’t forget to put the Guidance Scale back to how it was before (I always leave the sharpness at 10). I hope that in the future this can be simplified.

**Comment by usigg3** [comment]
Use bokeh or shallow depth of field in the negative prompt

**Comment by usigg3** [comment]
I don't really understand how this makes it more realistic. If you look at most good, real photos then you'll find that the background is out of focus.

**Comment by lllyasviel** [comment]
the best method is to write the content of background 
for example

with a car in the background
in front of a bookstore and street view 
in a classroom
etc

so if the model do not know what is in bg, the bg will be blur, even when you give blur in negative prompt
